### ISSUE TITLE
incident rollback with fast-rollback isn't updating the service selector in staging

### ISSUE THREAD

**Issue by erutherford** [InVisionApp/kit-deployer#87]
When running through an incident rollback the service selector wasn't updated in staging.

To compare to the results in production where `01db69651b98b608ea560a13550bdd1f9452cf0b` was the previously active deploy and `853f2d9767ef345baed013cab1a23b7db09afdb6` is what we were rolling back to.

Prod cluster output:
```
Service Selector Label:
id=853f2d9767ef345baed013cab1a23b7db09afdb6,name=test-rosie-v7-pod
```

Staging cluster output:
```
Service Selector Label:
id=01db69651b98b608ea560a13550bdd1f9452cf0b,name=test-rosie-v7-pod
```

**Comment by erutherford** [comment]
I've added this as a bug here, but after further thinking I think this may be a result of the way we handle incident rollbacks.  We currently don't rollback the code in staging, we ask the users to fix master then run through a deploy or end their deployment so in actuality this is no different than the current behavior.  I'm going to leave this ticket in place until we have a chance to discuss it during work hours.

**Comment by brianhall** [comment]
Closing this out after filing the following ticket - https://github.com/InVisionApp/project-xb500/issues/570. TL;DR - I think Rosie should rollback prod/staging at the same time.

**Comment by erutherford** [comment]
Sounds good to me, I believe that's what we're doing with Elroy too

**Comment by erutherford** [comment]
Sorry, this one should have stayed open @brianhall, but maybe reworded the other one was fine to close.  This is an actual bug that needs to be resolved.

**Comment by brianhall** [comment]
Ah, I see. It just so happens that it mimics the current behavior of Rosie. Re-opening and adding to our current Sprint.

**Comment by erutherford** [comment]
Actually, this belongs on kit-deployer and it's a slightly different issue.  I'll create a ticket in the right place with the right data and I'll add it to the current Sprint.  Sorry for the flip-flip/back and forth.
